EhPanda/View/Home/Toplists/ToplistsReducer.swift

```diff
@@ -146,9 +146,11 @@ struct ToplistsReducer: Reducer {
                 } else {
                     state.rawPageNumber[state.type]?.resetPages()
                 }
-                return ToplistsGalleriesRequest(catIndex: state.type.categoryIndex, pageNum: pageNum)
-                    .effect.map({ [type = state.type] in Action.fetchGalleriesDone(type, $0) })
-                    .cancellable(id: CancelID.fetchGalleries)
+                return .run { [type = state.type] send in
+                    let response = await ToplistsGalleriesRequest(catIndex: type.categoryIndex, pageNum: pageNum).response()
+                    await send(Action.fetchGalleriesDone(type, response))
+                }
+                .cancellable(id: CancelID.fetchGalleries)
```

Comment:
[nits]
There are warnings triggered by this kind of code.
Would you mind if you could install `SwiftLint` and build again?

Since it's not critical and not associated with the refactoring of Publisher-based Request, this PR will be approved anyway.